simulator/export/export.go

```diff
@@ -363,6 +371,30 @@ func (s *Service) listPcs(ctx context.Context, r *ResourcesForExport, eg *util.S
 	return nil
 }
 
+func (s *Service) listNamespaces(ctx context.Context, r *ResourcesForExport, eg *util.SemaphoredErrGroup, opts options) error {
+	if err := eg.Go(func() error {
+		nss, err := s.client.CoreV1().Namespaces().List(ctx, metav1.ListOptions{})
```

Comment:
This is a forward-looking implementation.
We would be deprecating some Resource services, and I didn't create the service like a `namespaceService`.